DESCRIPTION

```diff
@@ -47,6 +47,7 @@ Imports:
     survival
 Suggests:
     bibtex,
+    coin,
```

Comment:
what's coin? is partykit added?

Reply:
Its where the conditional testing used in ctree, etc. is implemented. I always got an error if it was not included. partykit is also in the suggests.

Reply:
cool thanks